src/pages/venues/[id]/map/index.tsx

```diff
@@ -0,0 +1,144 @@
+import { ApolloProvider, isApolloError, useQuery } from "@apollo/client";
+import { IconArrowLeft } from "hds-react";
+import { GetStaticPropsContext } from "next";
+import dynamic from "next/dynamic";
+import React from "react";
+
+import { VENUE_QUERY } from "..";
+import Page from "../../../../common/components/page/Page";
+import initializeCmsApollo from "../../../../domain/clients/cmsApolloClient";
+import initializeNextApiApolloClient, {
+  useNextApiApolloClient,
+} from "../../../../domain/clients/nextApiApolloClient";
+import Link from "../../../../domain/i18n/router/Link";
+import useRouter from "../../../../domain/i18n/router/useRouter";
+import { getLocaleOrError } from "../../../../domain/i18n/router/utils";
+import serverSideTranslationsWithCommon from "../../../../domain/i18n/serverSideTranslationsWithCommon";
+import { staticGenerationLogger } from "../../../../domain/logger";
+import styles from "./map.module.scss";
+
+const MapView = dynamic(
+  () => import("../../../../common/components/mapView/MapView"),
+  {
+    ssr: false,
+  }
+);
+
+const VenueMapPageContent: React.FC = () => {
+  const router = useRouter();
+  const { data, loading, error } = useQuery(VENUE_QUERY, {
+    variables: {
+      id: router.query.id,
+    },
+  });
+
+  if (loading || error) {
+    return null;
+  }
+
+  const location = data.venue?.position?.coordinates;
+  const title = data.venue?.name;
+  const venueId = data.venue?.id;
+  const venueMapItem = {
+    id: venueId,
+    title,
+    href: `/venues/tprek:${data.venue?.id}`,
+    location,
+  };
+  const backHref = {
+    pathname: "/venues/[id]",
+    query: {
+      id: venueId,
+    },
+  };
+
+  return (
+    <div className={styles.content}>
+      <div className={styles.mapHeader}>
+        <Link
+          href={backHref}
+          avoidEscaping
+          aria-label="Takaisin paikkanäkymään"
```

Comment:
Missing translation.

You may also need to apply this label on the anchor directly. NextJS may not propagate this prop.

Reply:
Yeah, I need to learn to use WP translations next :)